frontend/src/app/shared/components/time_entries/shared/modal/base.modal.html

```diff
@@ -18,12 +18,14 @@
   </div>
 
   <div class="op-modal--footer op-modal--footer_split-actions">
-    <button class="button -danger"
-            *ngIf="deleteAllowed"
-            (click)="destroy()"
-            [textContent]="text.delete"
-            [attr.title]="text.delete">
-    </button>
+    <section>
```

Comment:
This is semantically not a section: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section

> Sections should always have a heading, with very few exceptions.

I would just leave it a `div`